source/partials/drupal/dependencies-compatible.md

```diff
@@ -0,0 +1,29 @@
+---
+contenttype: [partial]
+categories: [--]
+newcms: [drupal, drupal8, drupal9]
+product: [--]
+integration: [--]
+tags: [--]
+reviewed: "2022-11-04"
+---
+
+You must confirm that your site is compatible with the latest version of Drupal before you change the Drupal core version.
+ 
+Review the steps below to confirm compatibility for your site's components.
+
```

Comment:
I'd recommend telling them to upgrade to the latest MariaDB in all environments before starting here. I think there's already a docs page for doing that somewhere.

Reply:
https://github.com/pantheon-systems/documentation/pull/8004/files#diff-7957c661cbb6ba9a2f1c80f9092a455c9a35e720c08c358aff2eb03ffb2da6e7R15 is about mariadb updates.

Reply:
i'm going to create a new issue for this, as the change is probably bigger than just adding it here.  Some guides have this page - https://pantheon.io/docs/guides/drupal-9-hosted-createbt/mariadb - and others don't.  I think they all need to be consistent - either remove this page and add the info to this partial, or add this page to all docs that use this partial.

Reply:
I take that back. updated this doc to include that info.